cachito/workers/pkg_managers/yarn.py

```diff
@@ -297,27 +347,27 @@ def _expand_replacements(nexus_replacements: Dict[str, dict]) -> Dict[str, dict]
     :param dict nexus_replacements: a dict of nexus replacements which may contain N:1 keys
     :return: a dict of nexus replacements where all N:1 keys have been expanded to N 1:1 keys
     """
-    expanded_replacements = {
+    expanded_yarn_lock_keys = {
         key: nexus_replacements[multi_key]
         for multi_key in nexus_replacements
         for key in map(str.strip, multi_key.split(","))
     }
-    return expanded_replacements
+    return expanded_yarn_lock_keys
 
 
 def _match_to_new_version(
-    dep_name: str, dep_version: str, expanded_replacements: Dict[str, dict]
+    dep_name: str, dep_version: str, expanded_yarn_lock_keys: Dict[str, dict]
```

Comment:
This rename, and many others below were probably unnecessary - this really is just a dict of Nexus replacements (a modified subset of the original yarn.lock), not the entire yarn.lock